index.html

```diff
@@ -24,25 +24,183 @@
               stroke-linejoin="round"
               stroke-width="2"
               d="M4 6h16M4 12h16M4 18h16"
-            ></path></svg
-        ></a>
+            ></path>
+          </svg>
+        </a>
       </nav>
     </header>
 
-    <section class="container">
-      <div class="content">
-        <div class="header2">
-          <h1>I'm Alusine, glad to meet you!</h1>
+    <main>
+      <section class="container bg-white">
+        <div class="content">
+          <div class="header2">
+            <h1>I'm Alusine, glad to meet you!</h1>
+          </div>
+          <div class="body2">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div class="conect"><p>LET’S CONNECT</p></div>
+          <div class="icons">
+            <ul>
+              <li>
+                <a href="#"><img src="images/Vector-1.svg" alt="twitter" /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/Linkedinicon.svg" alt="LinkedIn"
+                /></a>
+              </li>
+              <li>
+                <a href="#"><img src="images/Vector-2.svg" alt="Mark" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="images/Vector-3.svg" alt="Github" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="images/Vector.svg" alt="other" /></a>
+              </li>
+            </ul>
+          </div>
         </div>
-        <div class="body2">
+      </section>
+      <!-- End of first headline -->
+      <section class="works bg-white">
+        <div class="card">
+          <div class="card-image-container card-image"></div>
+          <div class="card-content">
+            <h3>Tonic</h3>
+            <div class="first-links">
+              <a href="#" class="canopy-style">CANOPY</a>
+              <!-- <img src="images/Counter.svg" alt="#" /> -->
+              <span class="canopy-image"></span>
+              <a href="#">Back End dev</a>
+              <!-- <img src="images/Counter.svg" alt="#" /> -->
+              <span class="back-end-image"></span>
+              <a href="#">2015</a>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="second-links">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <a href="#">See Project</a>
```

Comment:
- [ ] Following the Figma design, all the buttons have different behaviours on user action: `hover`. `active`, `pressed` and `disabled`. Kindly implement them from your CSS file to stick to the Figma design.

* Check out how they should look on each style:
![image](https://user-images.githubusercontent.com/80612925/176933374-d7d7f70b-6d42-4a78-a772-e52d68453378.png)